docs/model-dev-guide/dtrain/dtrain-implement.rst

```diff
@@ -78,20 +78,28 @@ Global Batch Size
 You can reduce computational overhead by setting the ``global_batch_size`` to the largest batch size
 that fits into a single GPU multiplied times the number of slots.
 
-During distributed training, the ``global_batch_size`` specified in the
-:ref:`experiment-config-reference` is partitioned across ``slots_per_trial`` GPUs. The per-GPU batch
-size is set to: ``global_batch_size // slots_per_trial``. If ``slots_per_trial`` does not divide
-``global_batch_size`` evenly, the remainder is dropped. For convenience, the per-GPU batch size can
-be accessed via the Trial API, using :func:`context.get_per_slot_batch_size
-<determined.TrialContext.get_per_slot_batch_size>`.
+.. note::
+
+   This feature only applies to :ref:`high-level-apis` (Trial APIs) and does not apply to the Core
+   API.
```

Comment:
@garrett361 please review

Reply:
I think something like the following might be a bit more precise: "The `global_batch_size` field will be automatically respected by the Trial APIs.  Core API users must reference this field explicitly and organize their code to respect its value in order to use this hyperparameter."

Reply:
thanks. resolved.